src/ObjectUtils/CoffFile.cpp

```diff
@@ -133,17 +131,14 @@ std::optional<SymbolInfo> CoffFileImpl::CreateSymbolInfo(
   symbol_info.set_demangled_name(llvm::demangle(name.get().str()));
   symbol_info.set_address(symbol_virtual_address);
 
-  // The COFF symbol table doesn't contain the size of symbols. Set a placeholder for now.
+  // The COFF symbol table doesn't contain the size of symbols. Set a placeholder which indicates
+  // that the size is unknown for now and try to deduce it later. We will later use that placeholder
+  // to know which sizes are unknown.
```

Comment:
The last two sentences basically say the same. I would change the last sentence to "We will later use that placeholder in DeduceDebugSymbolMissingSizesAsDistanceFromNextSymbol".

Reply:
Done. Thanks!